Code/GraphMol/Wrap/Mol.cpp

```diff
@@ -89,9 +89,11 @@ namespace RDKit {
   }
   PyObject *GetSubstructMatch(const ROMol &mol, const ROMol &query,bool useChirality=false,
                             bool useQueryQueryMatches=false){
-    NOGIL gil;
     MatchVectType matches;
-    SubstructMatch(mol,query,matches,true,useChirality,useQueryQueryMatches);
+    {
+      NOGIL gil;
```

Comment:
That's my plan.

----
Brian Kelley

> On Oct 14, 2015, at 12:25 AM, Greg Landrum <notifications@github.com> wrote:
> 
> In Code/GraphMol/Wrap/Mol.cpp:
> 
> >      MatchVectType matches;
> > -    SubstructMatch(mol,query,matches,true,useChirality,useQueryQueryMatches);
> > +    {
> > +      NOGIL gil;
> How about modifying the NOGIL definition so that it only does anything if the thread safety flag (RDK_BUILD_THREADSAFE_SSS) is set?
> 
> —
> Reply to this email directly or view it on GitHub.
>